### PR TITLE
Applied standardization to saras_memory script

### PR DESCRIPTION
**Pull request by iRagno**
[//]: # (**********************************)
[//]: # (** Fill in the following fields **)
[//]: # (**********************************)

[//]: # (Note: Lines beginning with syntax such as this one, are comments and will not be visible in your report!)

### Pull Request Prelude

[//]: # (Thank you for working on improving Hercules!)

[//]: # (Please complete these steps and check the following boxes by putting an `x` inside the brackets _before_ filing your Pull Request.)

- [x] I have followed [proper Hercules code styling][code].
- [x] I have read and understood the [contribution guidelines][cont] before making this PR.
- [x] I am aware that this PR will be closed if the above-mentioned criteria are not fulfilled.

### Changes Proposed

Applied a few changes to fit script with official one and applied standardization to entire script.

**Affected Branches:** `Master`

**Issues addressed:** `None`

### Known Issues and TODO List

[//]: # (Insert checklist here)
[//]: # (Syntax: - [ ] Checkbox)

[v]: # (**NOTE** Enable the setting "[v] Allow edits from maintainers." when creating your pull request if you have not already enabled it.)

[cont]: https://github.com/HerculesWS/Hercules/blob/master/CONTRIBUTING.md
[code]: https://github.com/HerculesWS/Hercules/wiki/Coding-Style

### CODE REVIEW

**Comment by HerculesWSAPI** [comment]
This change is [Reviewable](https://reviewable.io/reviews/herculesws/hercules/1884)

File: npc/re/instances/saras_memory.txt [hunk 1]
```diff
@@ -223,469 +223,468 @@ dali,138,118,0	script	Dimensional Device#dimen	PORTAL,{
...
+	cutin("sara_elder_irine1.bmp", 255);
 	mes("^FF0000As they wander off you hear strange voices from around the corner...^000000");
-	donpcevent instance_npcname("Doyen Irene#sarains")+"::OnDisable";
-	donpcevent instance_npcname("A girl#sarains")+"::OnDisable";
-	dispbottom(_("As they wander off you hear strange voices from around the corner...")),"";
-	disablenpc instance_npcname("Sara Irene#saratalk");
-	next;
+	donpcevent(instance_npcname("Doyen Irene#sarains")+"::OnDisable");
+	donpcevent(instance_npcname("A girl#sarains")+"::OnDisable");
+	dispbottom(_("As they wander off you hear strange voices from around the corner...")), "";
```
Comment (Asheraf) [inline npc/re/instances/saras_memory.txt]: This seems wrong :)

Comment (iRagno) [reply npc/re/instances/saras_memory.txt]: Done. Thank you Asheraf.

File: npc/re/instances/saras_memory.txt [hunk 2]
```diff
@@ -42,142 +42,142 @@ dali,130,107,5	script	Leon the Adventurer#Sara	4_M_DST_GRAND,{ // instance CD ch
 	mes("[Leon the Adventurer]");
 	if (BaseLevel < 99) {
 		mes("You know... this place doesn't seem to be safe for you. Please returnto me once you have achieved LV. 99");
```
Comment (Asheraf) [inline npc/re/instances/saras_memory.txt]: please correct this one `return to`

**Comment by Asheraf** [comment]
As for the changes in PR it looks good to me, but we should take into consideration merging the npc's with same functionality into one and using the duplicate function

File: npc/re/instances/saras_memory.txt (hunk 2 above)
Comment (iRagno) [reply npc/re/instances/saras_memory.txt]: Done. Just for the record, this was only focused on standardization, but I guess can also add typo fixes.

**Comment by iRagno** [comment]
Applied requested changes and squashed commits.

I saw the script can be improved in several parts, like duplicating npcs that are identical.. But, this was only fixing standardization to the script, and leaved any structural change for the future.

**Pull request merged**